tests/acceptance/testall

```diff
@@ -653,6 +658,8 @@ while true; do
       CF_SERVERD=${1#--cfserverd=};;
     --cfkey=*)
       CF_KEY=${1#--cfkey=};;
+    --cfrunagent=*)
+      CF_RUNAGENT=${1#--cfrunagent=};;
```

Comment:
+1. You should update the downstream testall scripts as well, to make sure this will work wherever we make an acceptance test:

* https://github.com/cfengine/enterprise/blob/master/tests/acceptance/testall#L65
* https://github.com/cfengine/masterfiles/blob/master/tests/acceptance/testall#L72